packages/fxa-settings/src/components/Settings/Modal/index.tsx

```diff
@@ -68,69 +68,66 @@ export const Modal = ({
           )}
           ref={modalInsideRef}
         >
-          <div
-            tabIndex={0}
-            ref={tabFenceRef}
-            data-testid="modal-tab-fence"
-            className="outline-none"
-          />
-          <div className="flex justify-end pe-2 py-2">
-            <button
-              data-testid="modal-dismiss"
-              onClick={(event) => onDismiss()}
-              title={ftlMsgResolver.getMsg('modal-close-title', 'Close')}
+          <div tabIndex={0} ref={tabFenceRef} data-testid="modal-tab-fence">
+            <div className="flex justify-end pe-2 py-2">
+              <button
+                className="border-2 border-transparent"
+                data-testid="modal-dismiss"
+                onClick={(event) => onDismiss()}
+                title={ftlMsgResolver.getMsg('modal-close-title', 'Close')}
```

Comment:
I think I'm a bit confused by what the.... oh... so above in a previous file we had:
```js
l10n.getString('alert-bar-close-message', null, 'Close message')
```

But here we don't have that `null` since this uses `ftlMsgResolver.getMsg()`, hence the slightly different syntax of:
```js
ftlMsgResolver.getMsg('modal-close-title', 'Close')
```

Reply:
Right, that's correct. We're converting to `FtlMsg` and `FtlMsgResolver` as we update components, but I missed that above. Thanks!